src/database/migrations/20241122084739-alter-entity-type-add-isExternal-key.js

```diff
@@ -0,0 +1,31 @@
+'use strict'
+
+module.exports = {
+	up: async (queryInterface, Sequelize) => {
+		// Add the `is_external` column to the `entity_types` table
+		await queryInterface.addColumn('entity_types', 'is_external', {
```

Comment:
@adithyadinesh0412 we can create a new column called config, in which all these 3 keys will be stored as JSON